pyocd/target/family/target_lpc5500.py

```diff
@@ -75,6 +83,29 @@ def _modify_ap1(self, seq):
         
         return seq
 
+    def check_locked_state(self, seq):
+        # Attempt to unlock cores if they are locked (flash is empty etc.)
+        self.was_core_locked = False
```

Comment:
Is `was_core_locked` used outside of this method? I mean, does it need to be an instance attribute?

Reply:
This has been removed